scripts/hangman.js

```diff
@@ -1012,7 +963,7 @@ function Hangman() {
                 u = game[1],
                 a = game[2].toUpperCase(),
                 h = game[3];
-                //c = game.length < 5 ? defaultParts : game[4];
+                // c = game.length < 5 ? defaultParts : game[4];

```

Comment:
i dont get why you're adding a space here. its just 1 more character to delete if that code is ever used again